app/src/main/java/de/westnordost/streetcomplete/data/AQuestDao.java

```diff
@@ -216,6 +216,12 @@ private int insertAll(Collection<T> quests, boolean replace)
 		return addRows;
 	}
 
+	public void clear()
+	{
+		SQLiteDatabase db = dbHelper.getReadableDatabase();
+		db.execSQL("DELETE FROM " + getTableName());
+	}
```

Comment:
So this, by the way, also deletes to-be-uploaded quests, reverted quests, ... everything.